src/main/java/com/dropbox/core/v2/DbxClientV2.java

```diff
@@ -171,7 +171,7 @@ public DbxRefreshResult refreshAccessToken() throws DbxException {
         @Override
         public DbxRefreshResult refreshAccessToken() throws DbxException {
             credential.refresh(this.getRequestConfig());
-            return new DbxRefreshResult(credential.getAccessToken(), credential.getExpiresAt());
+            return new DbxRefreshResult(credential.getAccessToken(), (credential.getExpiresAt() - System.currentTimeMillis())/1000);
```

Comment:
DbxRefreshResult expects "expires in" in seconds that means `(ExpiresAt (Millis) - currentMillis)/1000`